VOQC/client.py

```diff
@@ -0,0 +1,9 @@
+from voqc import *
+
+
+a = load("/home/ag/Documents/SQIR/VOQC/benchmarks/Arithmetic_and_Toffoli/tof_10.qasm")
+a.optimize()
+a.write("/home/ag/Documents/VOQC_PythonBindings/SQIR/VOQC/extraction/_build/with_qopt10.qasm")
```

Comment:
Hi @akshajgaur 

It is a good idea to avoid committing absolute paths in source code. Could you try to make these relative to the project path?

Reply:
Hi @k4rtik 

Thanks for the suggestion. I was in the process of changing the path to relative, and I think I have it done, so it should be in the next pull request. After this, it should now be relative and be able to be passed like this:

```
from voqc import *

a = load("benchmarks/Arithmetic_and_Toffoli/tof_3.qasm")
a.optimize()
a.write("extraction/tof.qasm")
```